BlueMap/Chat/src/main/resources/bluemap-chat.js

```diff
@@ -95,23 +152,60 @@ void async function() {
         }
     });
 
+    addEventListener("keydown", (e) => {
+            if (e.key === "t") {
+                toggleChat();
+                if (!chatExpanded) {
+                    toggleExpanded();
+                }
+            }
+        }
+    )
+
+    // Max message count. Check if it's set. If not, use 100
+    let mmc = "{{max-message-count}}"
+    // It is replaced from java side. To prevent this if statement from being replaced, a little hack is used.
+    if (mmc === "{{" + "max-message-count" + "}}") {
+        mmc = "100";
+    }
```

Comment:
Is there ever a case where it's not being replaced from java side? The java side already has default values for missing config options. Same for the web-chat-prefix.

Reply:
If there is an error, then I had a fallback

Reply:
Same thing as with the bmjs_moved comment, becomes unnecessary when bluemap-auth has been made optional.